app/test_main.py

```diff
@@ -1,3 +1,34 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat,dog,result",
+    [
+        (0, 0, [0, 0]),
+        (-27, 15, [0, 1]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
```

Comment:
i think you are suposed to have description of each test (see Testing/Pytest parametrize video,  on platform, start at 10:30)